tests/lib/rules/no-get.js

```diff
@@ -337,6 +337,22 @@ ruleTester.run('no-get', rule, {
         },
       ],
     },
+    {
+      // Nested path inside left side of assignment should not autofix.
+      // Optional chaining is not valid in the left side of an assignment.
+      code: "this.get('foo.bar')[123] = 'hello world';",
```

Comment:
In this case we could instead reasonably change it to `this.foo.bar[123]` instead. Would it be simple enough to do that fix instead of leaving it alone?

Reply:
That's a great point. I have updated to add back the autofix without optional chaining. Thanks!